### ISSUE TITLE
Searchable Orders with Foreign Key "File Manager" Column (Rinda)

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#1935]
As per Rinda's request over the weekend, she has an extremely useful idea - to add a searchable foreign key "FILE MANAGER" column in ORDERS (pulled from the "Tours" table)

It would primarily need to be searchable/filterable with order Month and Year (when it was created), but you might as well make is searchable/filterable with all entry boxes and drop-downs.

**Comment by chris206** [comment]
Can this be done quickly? it would be very helpful

**Comment by bradatkins** [comment]
Isn't that what this is?

![image](https://cloud.githubusercontent.com/assets/8440255/23389058/b912d158-fd33-11e6-846f-21a38e32103d.png)

**Comment by chris206** [comment]
That is a filterable/searchable foreign key "FILE MANAGER" column in BONUS. Very helpful to see who is handling an entire departure, totally unhelpful to determine who is responsible for new booking OR how many bookings were received in a given period of time.

**Comment by bradatkins** [comment]
Take a screenshot of what field in the admin you want to show up there. I'm
not sure which field you are talking about.

**Comment by bradatkins** [comment]
Oh you want the file manager in the orders screen?

**Comment by chris206** [comment]
yes, that is a much simpler way of saying it

**Comment by bradatkins** [comment]
Yes, that can be done. I noticed you were using words like foreign key, table, etc.

Just keep it to the requirements and the message will be clearer.

Close this issue and log a separate GH which will make it clear and we can do it as an enhancement.

**Comment by chris206** [comment]
I just want to make it really, really specific to avoid making changes to Orders but NOT making changes to Waitlisted Orders; or building multiple places for conflicting info

**Comment by bradatkins** [comment]
Oh it's all the same screen actually :)
